.github/workflows/scheduled-tests.yml

```diff
@@ -59,8 +59,8 @@ jobs:
         uses: ./.github/workflows/_integration-tests.yml
         with:
             packages: ${{ toJSON('[dbt-athena, dbt-bigquery, dbt-postgres, dbt-redshift, dbt-snowflake, dbt-spark]') }}
-            branch: ${{ github.event.pull_request.head.ref }}
-            repository: ${{ github.event.pull_request.head.repo.full_name }}
+            branch: ${{ github.ref }}
+            repository: ${{ github.repository }}
```

Comment:
Will this work on forks?

Reply:
this is just scheduled tests.  there is no pull request event.  this was coming through as empty so was just defaulting back which is why there was no error.  but it means when you ran this off a branch it was still running off main.